pincer/cog.py

```diff
@@ -0,0 +1,114 @@
+# Copyright Pincer 2021-Present
+# Full MIT License can be found in `LICENSE` at the project root.
+
+from __future__ import annotations
+from asyncio import ensure_future
+
+from importlib import reload, import_module
+from types import ModuleType
+from typing import TYPE_CHECKING
+
+from . import client as _client
```

Comment:
Would this cause a circular import?

Reply:
I don't think it will. Module imports don't usually do that.